### DISCUSSION TITLE
🥳 LobeChat Cloud Comming Soon, Join Our Waitlist | SaaS 版本即将发布，快来加入愿望单

### DISCUSSION THREAD

**Discussion by canisminor1990** [lobehub/.github#13] [Announcements]
[![](https://github.com/lobehub/lobe-chat/assets/17870709/56a44a54-580d-459a-a9e8-de7c598352a0)](https://lobehub.com?utm_source=Github&utm_medium=discussion-1390&utm_campaign=image&utm_id=saas-cloud)

Hi everyone! our SaaS product LobeChat Cloud are coming soon! 🥳
If you are interested with it, please come here to join our waitlist. ⛳️  Just click here 👉 [lobehub.com](https://lobehub.com?utm_source=Github&utm_medium=discussion-1390&utm_campaign=english&utm_id=saas-cloud)

You can find all the details about what LobeChat Cloud on it. Don't forget to join our waitlist for the latest news. Can't wait for you to check it out!

We will gradually start Early Access from the waitlist in mid to late April. Please stay tuned~ 🦜

---

大家好！伴期待已久的 SaaS 版本 LobeChat Cloud 即将发布啦！🥳
如果您对此感兴趣，请来这里加入我们的愿望单单。⛳️ 点击这里 👉 [lobehub.com](https://lobehub.com?utm_source=Github&utm_medium=discussion-1390&utm_campaign=chinese&utm_id=saas-cloud)

你可以在上面找到所有关于 LobeChat Cloud 的详细信息。别忘了加入我们的等待名单，以获取最新消息。迫不及待想让你们去看看！

我们会在四月中下旬陆续从 Waitlist 中开启 Early Access 敬请期待~ 🦜

https://github.com/lobehub/lobe-chat/assets/134299653/21a9035a-d89a-427e-96ed-d616da6ad790

**Comment by mushan0x0** [comment]
官网泰酷辣😍

**Comment by Osub** [comment]
漂亮

**Comment by Alitarong** [comment]
仰慕大佬已久，能加一个联系方式交流一下技术吗？能受累加一下您微信吗？ 13820210795（qiaoyu liu）
